packages/wagmi-core/src/alchemySmartWallet.ts

```diff
@@ -18,7 +18,7 @@ import {
   type Client,
   createClient,
 } from "viem";
-import { createConnector, type CreateConnectorFn } from "wagmi";
+import { createConnector, type CreateConnectorFn } from "@wagmi/core";
```

Comment:
`"wagmi"` a react-specific pkg, so we shouldn't use it in our `"@alchemy/wagmi-core"`